ggml-cuda.cu

```diff
@@ -512,6 +512,13 @@ static size_t g_scratch_offset = 0;
 
 static cublasHandle_t g_cublas_handles[GGML_CUDA_MAX_DEVICES] = {nullptr};
 
+static __device__ void bad_arch() {
```

Comment:
```suggestion
[[noreturn]]
static __device__ void bad_arch() {
```
And remove the `return` statements after a call to bad_arch, assuming nvcc is smart enough...

Reply:
Seems to work.